portfolio/src/main/java/com/google/sps/servlets/ListComments.java

```diff
@@ -0,0 +1,63 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
```

Comment:
So a few things on licenses:

* Just because you use Java does not mean YOUR code needs to have the Apache license
* Depending on how open source you want your code to be, some flavor of a gnu public license might be more appropriate
* Generally you put your license once in the readme instead of in each code header (otherwise it clutters things up to have a long header in a bunch of places)

Let me know if you have questions about licensing and that whole world.

Reply:
I see. Those line of licenses documentation was in the file originally, so I didn't know that it should be deleted. I have deleted those licenses.